<launch>
	<arg name="tf_prefix" value="upper_frames" />

	<node pkg="openni_tracker_modified" name="openni_tracker" type="openni_tracker" output="screen">
		<param name="tf_prefix" value="$(arg tf_prefix)" />
		<param name="fixed_frame" value="openni_depth_frame" />
	</node>


	<node pkg="users_markers" name="users_markers" type="users_markers" output="screen">
		<rosparam file="$(find users_markers)/params/marker_params.yaml" command="load" />
		<param name="tf_prefix" value="$(arg tf_prefix)" />
		<param name="fixed_frame" value="openni_depth_frame" />
	</node>

</launch>
